modular_zubbers/modules/jobs/job_types/blacksmith.dm

```diff
@@ -0,0 +1,61 @@
+/datum/job/blacksmith
+	title = JOB_BLACKSMITH
+	description = "Smith wares, Sell them."
+	department_head = list(JOB_HEAD_OF_PERSONNEL)
+	faction = FACTION_STATION
+	total_positions = 1
+	spawn_positions = 1
+	supervisors = SUPERVISOR_HOP
+	config_tag = "BLACKSMITH"
+	exp_granted_type = EXP_TYPE_CREW
+
+	outfit = /datum/outfit/job/blacksmith
+	plasmaman_outfit = /datum/outfit/plasmaman/curator //Not sure how this works. Leaving it alone.
+
+	paycheck = PAYCHECK_CREW
+	paycheck_department = ACCOUNT_SRV
+
+	display_order = JOB_DISPLAY_ORDER_BLACKSMITH
+	departments_list = list(
+		/datum/job_department/service,
+		)
+
+	mail_goodies = list(
+		/obj/item/book/random = 44,
+		/obj/item/book/manual/random = 5,
+		/obj/item/book/granter/action/spell/blind/wgw = 1,
+	)
+
+	family_heirlooms = list(/obj/item/pen/fountain, /obj/item/storage/dice)
+
+	job_flags = JOB_ANNOUNCE_ARRIVAL | JOB_CREW_MANIFEST | JOB_EQUIP_RANK | JOB_CREW_MEMBER | JOB_NEW_PLAYER_JOINABLE | JOB_REOPEN_ON_ROUNDSTART_LOSS | JOB_ASSIGN_QUIRKS | JOB_CAN_BE_INTERN
+
+	voice_of_god_silence_power = 3
+	rpg_title = "Ye olde Smithy"
+
+/datum/outfit/job/blacksmith
+	name = "Blacksmith"
+	jobtype = /datum/job/blacksmith
+
+	id_trim = /datum/id_trim/job/curator
+	uniform = /obj/item/clothing/under/rank/civilian/curator
```

Comment:
Some outfit for them would be appreciated, perhaps an apron, but they're not a necessity! for this PR either!

Reply:
same deal with this, this WILL be changed. these are placeholders

Reply:
Resolved  in the Next Push. Now have Cargo pants, brown gloves, and boots.